BraveRewardsUI/Settings/Ads/Details/AdsDetailsViewController.swift

```diff
@@ -43,9 +43,23 @@ class AdsDetailsViewController: UIViewController {
     
     title = Strings.settingsAdsTitle
     
+    generateRows()
+
     fetchAdsDetails()
   }
   
+  func generateRows() {
+    rows.removeAll()
+
+    rows.append(.adsPerHour)
+    if state.ads.shouldAllowSubdivisionTargeting {
+      rows.append(.subdivisionTargeting)
+    }
```

Comment:
Is `shouldAllowSubdivisionTargeting` always evaluated from ads? There's no way for a user to toggle this on and off right?

Reply:
This is always evaluated from ads